schemas/s51-advice.schema.json

```diff
@@ -45,6 +45,10 @@
 			"description": "Title of the advice",
 			"examples": ["Advice following issue of decision to accept the application for examination"]
 		},
+		"titleWelsh": {
+			"type": "string",
```

Comment:
Just need to support null here, then looks good.

Reply:
Ah yeah! That's fixed :)